tests/pems/mixed_chains/generate-certs.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Usage: ./generate_certs.sh [clean]
+# Generates all necessary certs for benching
```

Comment:
"testing" might be more accurate? And it's not really "all necessary" certs, just the mixed chain certs.
```suggestion
# Generates mixed-chain certs for testing
```